docs/DeveloperGuide.md

```diff
@@ -156,44 +156,60 @@ This section describes some noteworthy details on how certain features are imple
 
 ### Add Item feature
 
+The add item feature allows users to add an `InventoryItem` to be tracked by the system. 
+
 #### Implementation
 
 The add item command will be executed by `AddItemCommand`. Items added will be stored in `InventoryList`.
 
 Given below is an example usage scenario and how the add item mechanism behaves at each step.
 
-Step 1. The user launches the application for the first time. The `TrackO` will be initialized with the initial TrackO state, and the `InventoryList` will contain sample data.
+Step 1. The user launches the application for the first time. The `TrackO` will be initialized with the initial TrackO state, and the `InventoryList` will contain sample data. <br>
+The initial state of TrackO Model before running add item command will be as such.
+
+![AddItemState0](images/developer-guide/AddItemState0.png)
+
+Step 2. The user executes `addi i/Key q/200 d/Cabinet keys sp/9.90 cp/4.90 t/new` command to add 200 keys to item list in TrackO. 
+The `addi` command creates an `AddItemCommandParser` which checks the necessary input arguments for 
+* item name (prefixed by `i/`)
+* quantity (prefixed by `q/`) 
+* description (prefixed by `d/`)
+* sell price (prefixed by `sp/`)
+* cost price (prefied by `cp`)
 
-Step 2. The user executes `addi i/keys q/10 d/Metallic keys sp/15.50 cp/10 t/Utilities` command to add 10 `keys` to item list in TrackO.
-The `addi` command creates an `AddItemCommandParser` which checks the necessary input arguments for item name (prefixed by `i/`), 
-quantity (prefixed by `q/`), description (prefixed by `d/`), selling price (prefixed by `sp/`), cost price (prefixed by `cp/`) 
-are present before parsing the arguments into an `AddItemCommand` object. Tags, prefixed by `t/`, are optional.
+are present before parsing the arguments into an `AddItemCommand` object. 
```

Comment:
Here it feels a bit awkward that the sentence is being obstructed by the points. Maybe can try something like

```
The `addi` command creates an `AddItemCommandParser` which checks that the following input arguments are present before parsing the arguments into an `AddItemCommand` object:
* item name (prefixed by `i/`)
* quantity (prefixed by `q/`) 
* description (prefixed by `d/`)
* sell price (prefixed by `sp/`)
* cost price (prefied by `cp`)
```

Reply:
ah thanks for the suggestion! I've made the changes